src/js/components/Filter/FilterForm.jsx

```diff
@@ -130,14 +131,15 @@ const FilterForm = ({
                       label="react.button.clear.label"
                       onClick={() => onClearHandler(form)}
                       variant="transparent"
-                      type="submit"
+                      type="button"
                     />
                     <Button
                       defaultLabel="Search"
                       label="react.button.search.label"
+                      onClick={() => updateFilterParams(values)}
                       disabled={!allowEmptySubmit && _.every(values, value => !value)}
                       variant="primary"
-                      type="submit"
+                      type="button"
```

Comment:
I'm just wondering why it has been changed? Isn't `onSubmit` unnecessary now?
What were the sideeffects that `onClick={() => updateFilterParams(values)}` handles and submit doesn't, if `onSubmit` we are also calling the same (`updateFilterParams`)?